pkg/snowflake/task.go

```diff
@@ -128,7 +128,7 @@ func (tb *TaskBuilder) Create() string {
 		sort.Strings(sortedKeys)
 
 		for _, k := range sortedKeys {
-			sp = append(sp, EscapeString(fmt.Sprintf(`%v = %v`, k, tb.session_parameters[k])))
+			sp = append(sp, fmt.Sprintf(`%v = %v`, k, tb.session_parameters[k]))
```

Comment:
I think you also need to wrap the value in quotes since some of those values aren't valid ids

Reply:
It's difficult here because in the case of `CLIENT_TIMESTAMP_TYPE_MAPPING = TIMESTAMP_LTZ` for example, snowflake won't recognize that if you quote it? I figured I'd leave it up to the end user here to make sure they quote what needs to be quotes?

Reply:
It will as long as they all caps it

Reply:
Since most of the other ids in teh system are auto quoted in the provider I would prefer just quoting everything